README.md

```diff
@@ -107,6 +107,7 @@ The spider `download.py`, dumps the response body as unicode to the files. The l
 	  -s, --set TEXT    Settings to be passed to the Scrapy command. Use with the
 	                    bookworm/broadworm commands.
 
+	  --random_payload_size INTEGER  Add a random payload with the given size.
```

Comment:
I am on a fence here. @lopuhin should we add a special option or we can refer to `-s SCRAPY_BENCH_RANDOM_PAYLOAD_SIZE=x` here?

Reply:
I have a slight preference to use scrapy settings here, documenting all "interesting" settings in the README, as they require less effort to implement and it's more clear that they are per-spider.

Reply:
Okay I'll change it then.